packages/experimental/ExperimentalIcon/src/__tests__/Icon.test.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import { SvgIcon } from '../';
+import * as renderer from 'react-test-renderer';
+
+import { checkRenderConsistency, checkReRender } from '@fluentui-react-native/test-tools';
+
+describe('Icon component tests', () => {
+  it('SVG Icon', () => {
+    const tree = renderer.create(<SvgIcon viewBox="0 0 1000 1000" />).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('Icon checkRenderConsistency', () => {
+    checkRenderConsistency(() => <SvgIcon />, 2);
```

Comment:
I don't think these tests are particularly useful if svg information isn't being passed in

Reply:
These tests were, also, created automatically. There will be separate task for automation

Reply:
I used FontIcon with props for now. I'll write SVG tests in a separate task